packages/hebao_v1/contracts/stores/SecurityStore.sol

```diff
@@ -65,15 +82,20 @@ contract SecurityStore is DataStore
         return wallets[wallet].guardians.length;
     }
 
-    function addGuardian(address wallet, address guardian)
+    function addGuardian(address wallet, address guardian, uint info)
```

Comment:
should we check `info != 0`

Reply:
I would definitely not do it in the storage (storage shouldn't need to care), and even in the module I wouldn't do it because it's valid and may be useful one day.